include/fi_mem.h

```diff
@@ -312,6 +312,9 @@ static inline void *util_buf_alloc_ex(struct util_buf_pool *pool, void **context
 	struct util_buf_footer *buf_ftr;
 
 	buf = util_buf_alloc(pool);
+	if (!buf)
```

Comment:
`if OFI_UNLIKELY(!buf)` ? this is micro-optimization, but it can help compiler to force branch prediction a more popular way.